auth.cfg

```diff
@@ -0,0 +1,2 @@
+[google]
+key = YOUR_KEY_HERE
```

Comment:
I agree that pip installs make including cfg files a bad idea.

On the other hand, we have run into this problem of configuration before, some of which can get quite complicated (logging conf, server conf, auth conf -- which could be arbitrarily complex for certain sites) and I don't think "add another command line flag" is the solution to all of them.  To me, it makes more sense to have _some_ file(s) somewhere that, yes, a user can point to with a command line argument.

The "pre-populate configs in some .py file and have a cmd line flag to select from amongst them" solution doesn't work in cases where, like auth, we do not know the values in advance.

I'll add a command line flag for this for the time being, but we really need a better solution, IMHO.